addons/point_of_sale/static/src/js/devices.js

```diff
@@ -248,16 +252,17 @@ var ProxyDevice  = core.Class.extend(mixins.PropertiesMixin,{
     // try several time to connect to a known proxy url
     try_hard_to_connect: function(url,options){
         options   = options || {};
-        var port  = ':' + (options.port || '8069');
+        var protocol = window.location.protocol;
```

Comment:
shouldn't we look also at the url, if the person explicitly write "http://10.0..."

Reply:
We ask for an IP address, so this would be a little strange I think. It was like this before and I cannot find any tickets about it in support.
![screenshot at 2018-10-17 15-55-08](https://user-images.githubusercontent.com/8284204/47091408-1b649680-d225-11e8-8068-317ee7d2b77f.png)

If you are thinking about people forcing their pos to use a http pos-box they would be out of luck, because the browser will not allow them to do this.